### ISSUE TITLE
Failing on RedHat

### ISSUE THREAD

**Issue by jschaeff** [camptocamp/puppet-dell#63]
There is another fix to make this plugin work on RedHat/CentOS.

The error you get is : 

```
Error: Could not update: Execution of '/usr/bin/yum -d 0 -e 0 -y install firmware-addon-dell' returned 1: Error: Package: python-smbios-2.2.27-4.4.1.el6.x86_64 (dell-omsa-specific)
           Requires: libsmbios = 2.2.27-4.4.1.el6
           Installed: libsmbios-2.2.27-1739.8348.el6.x86_64 (@dell-system-update_dependent)
               libsmbios = 2.2.27-1739.8348.el6
           Available: libsmbios-2.2.26-6.1.el6.x86_64 (dell-omsa-indep)
               libsmbios = 2.2.26-6.1.el6
           Available: libsmbios-2.2.27-4.4.1.el6.x86_64 (dell-omsa-specific)
               libsmbios = 2.2.27-4.4.1.el6
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
Error: /Stage[main]/Dell::Openmanage::Redhat/Package[firmware-addon-dell]/ensure: change from absent to latest failed: Could not update: Execution of '/usr/bin/yum -d 0 -e 0 -y install firmware-addon-dell' returned 1: Error: Package: python-smbios-2.2.27-4.4.1.el6.x86_64 (dell-omsa-specific)
           Requires: libsmbios = 2.2.27-4.4.1.el6
           Installed: libsmbios-2.2.27-1739.8348.el6.x86_64 (@dell-system-update_dependent)
               libsmbios = 2.2.27-1739.8348.el6
           Available: libsmbios-2.2.26-6.1.el6.x86_64 (dell-omsa-indep)
               libsmbios = 2.2.26-6.1.el6
           Available: libsmbios-2.2.27-4.4.1.el6.x86_64 (dell-omsa-specific)
               libsmbios = 2.2.27-4.4.1.el6
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
```

I had to comment the "package{'firmware-addon-dell':}" section to make the module work.

**Comment by raphink** [comment]
Which version of RedHat is that?

**Comment by jschaeff** [comment]
I had the issue on CentOS7.2 and 6.7

**Comment by D43m0n** [comment]
The issue persists with the 0.6.1 release of the module.

Repo's from dell are accessible, it seems to fail here:

```
Error: Could not update: Execution of '/bin/yum -d 0 -e 0 -y list firmware-tools' returned 1: Error: No matching Packages to list
Error: /Stage[main]/Dell::Hwtools::Package/Package[firmware-tools]/ensure: change from absent to latest failed: Could not update: Execution of '/bin/yum -d 0 -e 0 -y list firmware-tools' returned 1: Error: No matching Packages to list
Error: Could not update: Execution of '/bin/yum -d 0 -e 0 -y list smbios-utils' returned 1: Error: No matching Packages to list
Error: /Stage[main]/Dell::Hwtools::Package/Package[smbios-utils]/ensure: change from absent to latest failed: Could not update: Execution of '/bin/yum -d 0 -e 0 -y list smbios-utils' returned 1: Error: No matching Packages to list
```

Running on RHEL7.3

**Comment by raphink** [comment]
This should have been fixed by #76.
